app/controllers/user/sessions_controller.rb

```diff
@@ -8,8 +8,8 @@ class User::SessionsController < Devise::SessionsController
   def new
     super do |resource|
       if params[:commit].present?
-        resource.errors.add(:email, :invalid_authentication)
-        resource.errors.add(:password, :invalid_authentication)
+        resource.valid?
```

Comment:
you can use `resource.validate` here, is an alias for `valid?` but is more semantic in this case